src/Umpirsky/ListGenerator/Exporter/SimpleXMLExtended.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Umpirsky\ListGenerator\Exporter;
+
+class SimpleXMLExtended extends \SimpleXMLElement
+{
+    public function addCData($text)
+    {
+        $node = dom_import_simplexml($this);
+        $node->appendChild(
+            $node->ownerDocument->createCDATASection($text)
+        );
+        return $node; // ??
```

Comment:
Why did you add `return $node;`?

Reply:
Likely because I copied it from somewhere...